db/seeds.rb

```diff
@@ -1,7 +1,7 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+
+User.find_or_create_by(email: 'admin@example.com') do |user|
+  user.password = 'password'
+end
```

Comment:
A really simple seed just to get us going. We can add something a little more sophisticated later.

With this in place, when you run `rails db:seed` an admin user will be created (if they do not already exist).